samples/sampler/stories/default/PageViews.js

```diff
@@ -8,6 +8,7 @@ PageViews.displayName = 'PageViews';
 
 const Config = mergeComponentMetadata('PageViews', PageViews);
 const propOptions = {
+	pageIndicatorPosition: ['top', 'bottom'],
```

Comment:
Let's add this control when this prop became public.

Reply:
It's removed.